### PR TITLE
Add default user callback for message entity 'uid'.

### PR DESCRIPTION
**Pull request by mccrodp**
Rerolled #107.

May still fail due to: https://github.com/Gizra/message/issues/108

### CODE REVIEW

File: src/Tests/MessageCreateTest.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  /**
+   * Tests if message create sets the default uid to currently logged in user.
+   */
+  public function testMessageCreateDefaultValues() {
+    // Login our user to create message.
+    $this->drupalLogin($this->user);
+
+    $type = 'dummy_message';
+    // Create message to be rendered without setting owner.
+    $message_type = $this->createMessageType($type, 'Dummy message', '', ['[message:author:name]']);
```
Comment (amitaibu) [inline src/Tests/MessageCreateTest.php]: `createMessageTemplate`? 😉

Comment (mccrodp) [inline src/Tests/MessageCreateTest.php]: Thanks!!

**Comment by mccrodp** [comment]
So it's still failing at the same point:

```
Fail      Other      MessageCreateTest   51 Drupal\message\Tests\MessageCreateT
    The default value for uid was set correctly.
    Value &#039;2&#039; is equal to value NULL.
```

I'm not sure what `&#039;2&#039;` is, so I will check this week. It should not be `NULL` anyway, so it looks like this release blocking issue is still valid: https://github.com/Gizra/message/issues/108

File: src/Tests/MessageCreateTest.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    $template = 'dummy_message';
+    // Create message to be rendered without setting owner.
+    $message_template = $this->createMessageTemplate($template, 'Dummy message', '', ['[message:author:name]']);
+    $message = Message::create(['template' => $message_template->id()]);
+
+    $message->save();
+
+    /* @var Message $message */
+    $this->assertEqual($this->user->id(), $message->getOwnerId(), 'The default value for uid was set correctly.');
```
Comment (jhedstrom) [inline src/Tests/MessageCreateTest.php:51]: This failing test is fixed by #130, so if this PR is merged at the same time as that one, all will go green =)

**Comment by amitaibu** [comment]
Merged based on @jhedstrom comment.

**Pull request merged**